----------------------------------------------------------------------
Timestamp - 21/04/2021 09:44:32 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 6084
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 10:01:11 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 6084
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 10:07:11 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 6084
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 10:14:33 +03:00 | Level - Error |
----------------------------------------------------------------------
RequestId : "0HM84I3GBUJM0:00000121"
RequestPath : "/admin"
SpanId : "|90264ff3-4c6489906d99a22e."
TraceId : "90264ff3-4c6489906d99a22e"
ParentId : ""
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 6084
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 10:15:52 +03:00 | Level - Error |
----------------------------------------------------------------------
Error : "Value cannot be null. (Parameter 'name')"
StackTrace : "   at System.Globalization.RegionInfo..ctor(String name)
   at CookieService.CookieSvc.GetUserCountry() in C:\TFS\identity windo\CMS_CORE_NG\CookieService\CookieSvc.cs:line 90"
InnerException : null
Source : "System.Private.CoreLib"
ActionId : "27989ba7-9397-4412-9ac1-ecda597ac2ec"
ActionName : "CMS_CORE_NG.Areas.Admin.Controllers.AccountController.Login (CMS_CORE_NG)"
RequestId : "0HM84I3GBUJM0:0000013B"
RequestPath : "/Admin/Account/Login"
SpanId : "|90265000-4c6489906d99a22e."
TraceId : "90265000-4c6489906d99a22e"
ParentId : ""
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 6084
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 10:26:26 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 10:31:32 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 10:31:34 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 10:31:50 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 10:37:37 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 10:42:29 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 10:51:12 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 10:51:13 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 10:51:19 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 10:51:27 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 10:51:28 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 10:52:34 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 10:52:40 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 10:52:44 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 10:52:46 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 11:11:59 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 11:11:59 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 11:11:59 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 11:11:59 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 11:11:59 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 11:12:10 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 11:12:10 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 11:14:01 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 11:17:55 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 11:17:58 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 11:18:01 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 11:18:03 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 11:18:06 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 11:18:12 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 11:18:17 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 11:21:59 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 11:22:03 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 11:26:56 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 11:27:02 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 11:28:46 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 11:57:08 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 11:57:09 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 11:57:19 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 12:04:24 +03:00 | Level - Error |
----------------------------------------------------------------------
database : "CmsCoreNg_DEV"
server : "."
EventId : { Id: 20004, Name: "Microsoft.EntityFrameworkCore.Database.Connection.ConnectionError" }
SourceContext : "Microsoft.EntityFrameworkCore.Database.Connection"
ActionId : "66977eac-6dc8-4ec8-8841-91a442e5da46"
ActionName : "CMS_CORE_NG.Controllers.Api.v1.CountryController.GetCountries (CMS_CORE_NG)"
RequestId : "0HM84IQTJN23E:0000010D"
RequestPath : "/api/v1/country/getcountries"
SpanId : "|8954f903-4b6d7db69a6ff0c8."
TraceId : "8954f903-4b6d7db69a6ff0c8"
ParentId : ""
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 12:04:24 +03:00 | Level - Error |
----------------------------------------------------------------------
contextType : "DataService.ApplicationDbContext"
newline : "
"
error : "Microsoft.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)
 ---> System.ComponentModel.Win32Exception (2): The system cannot find the file specified.
   at Microsoft.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, Boolean applyTransientFaultHandling, String accessToken, DbConnectionPool pool, SqlAuthenticationProviderManager sqlAuthProviderManager)
   at Microsoft.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnectionAsync(Boolean errorsExpected, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnectionAsync(Boolean errorsExpected, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenAsync(CancellationToken cancellationToken, Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:2,State:0,Class:20"
EventId : { Id: 10100, Name: "Microsoft.EntityFrameworkCore.Query.QueryIterationFailed" }
SourceContext : "Microsoft.EntityFrameworkCore.Query"
ActionId : "66977eac-6dc8-4ec8-8841-91a442e5da46"
ActionName : "CMS_CORE_NG.Controllers.Api.v1.CountryController.GetCountries (CMS_CORE_NG)"
RequestId : "0HM84IQTJN23E:0000010D"
RequestPath : "/api/v1/country/getcountries"
SpanId : "|8954f903-4b6d7db69a6ff0c8."
TraceId : "8954f903-4b6d7db69a6ff0c8"
ParentId : ""
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
----------------------EXCEPTION DETAILS-------------------------------
Exception - Microsoft.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)
 ---> System.ComponentModel.Win32Exception (2): The system cannot find the file specified.
   at Microsoft.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, Boolean applyTransientFaultHandling, String accessToken, DbConnectionPool pool, SqlAuthenticationProviderManager sqlAuthProviderManager)
   at Microsoft.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnectionAsync(Boolean errorsExpected, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnectionAsync(Boolean errorsExpected, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenAsync(CancellationToken cancellationToken, Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:2,State:0,Class:20StackTrace -    at Microsoft.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, Boolean applyTransientFaultHandling, String accessToken, DbConnectionPool pool, SqlAuthenticationProviderManager sqlAuthProviderManager)
   at Microsoft.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnectionAsync(Boolean errorsExpected, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnectionAsync(Boolean errorsExpected, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenAsync(CancellationToken cancellationToken, Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()Message - A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)Source - Core Microsoft SqlClient Data ProviderInnerException - System.ComponentModel.Win32Exception (2): The system cannot find the file specified.---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 12:04:24 +03:00 | Level - Error |
----------------------------------------------------------------------
EventId : { Id: 1, Name: "UnhandledException" }
SourceContext : "Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware"
RequestId : "0HM84IQTJN23E:0000010D"
RequestPath : "/api/v1/country/getcountries"
SpanId : "|8954f903-4b6d7db69a6ff0c8."
TraceId : "8954f903-4b6d7db69a6ff0c8"
ParentId : ""
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
----------------------EXCEPTION DETAILS-------------------------------
Exception - Microsoft.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)
 ---> System.ComponentModel.Win32Exception (2): The system cannot find the file specified.
   at Microsoft.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, Boolean applyTransientFaultHandling, String accessToken, DbConnectionPool pool, SqlAuthenticationProviderManager sqlAuthProviderManager)
   at Microsoft.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnectionAsync(Boolean errorsExpected, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnectionAsync(Boolean errorsExpected, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenAsync(CancellationToken cancellationToken, Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()
   at Microsoft.EntityFrameworkCore.EntityFrameworkQueryableExtensions.ToListAsync[TSource](IQueryable`1 source, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.EntityFrameworkQueryableExtensions.ToListAsync[TSource](IQueryable`1 source, CancellationToken cancellationToken)
   at CMS_CORE_NG.Controllers.Api.v1.CountryController.GetCountries() in C:\TFS\identity windo\CMS_CORE_NG\CMS_CORE_NG\Controllers\Api\v1\CountryController.cs:line 29
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:2,State:0,Class:20StackTrace -    at Microsoft.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, Boolean applyTransientFaultHandling, String accessToken, DbConnectionPool pool, SqlAuthenticationProviderManager sqlAuthProviderManager)
   at Microsoft.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnectionAsync(Boolean errorsExpected, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnectionAsync(Boolean errorsExpected, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenAsync(CancellationToken cancellationToken, Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()
   at Microsoft.EntityFrameworkCore.EntityFrameworkQueryableExtensions.ToListAsync[TSource](IQueryable`1 source, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.EntityFrameworkQueryableExtensions.ToListAsync[TSource](IQueryable`1 source, CancellationToken cancellationToken)
   at CMS_CORE_NG.Controllers.Api.v1.CountryController.GetCountries() in C:\TFS\identity windo\CMS_CORE_NG\CMS_CORE_NG\Controllers\Api\v1\CountryController.cs:line 29
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)Message - A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)Source - Core Microsoft SqlClient Data ProviderInnerException - System.ComponentModel.Win32Exception (2): The system cannot find the file specified.---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 12:06:34 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 12:06:34 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 12:06:34 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 12:06:34 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 12:06:34 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 12:06:34 +03:00 | Level - Error |
----------------------------------------------------------------------
EventId : { Id: 1, Name: "UnhandledException" }
SourceContext : "Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware"
RequestId : "0HM84IQTJN23E:00000165"
RequestPath : "/polyfills.js"
SpanId : "|8954f94b-4b6d7db69a6ff0c8."
TraceId : "8954f94b-4b6d7db69a6ff0c8"
ParentId : ""
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
----------------------EXCEPTION DETAILS-------------------------------
Exception - System.Net.Http.HttpRequestException: Failed to proxy the request to http://localhost:64809/polyfills.js, because the request to the proxy target failed. Check that the proxy target server is running and accepting requests to http://localhost:64809/.

The underlying exception message was 'An error occurred while sending the request.'.Check the InnerException for more details.
 ---> System.Net.Http.HttpRequestException: An error occurred while sending the request.
 ---> System.IO.IOException: Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host..
 ---> System.Net.Sockets.SocketException (10054): An existing connection was forcibly closed by the remote host.
   --- End of inner exception stack trace ---
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.GetResult(Int16 token)
   at System.Net.Http.HttpConnection.FillAsync()
   at System.Net.Http.HttpConnection.ReadNextResponseHeaderLineAsync(Boolean foldedHeadersAllowed)
   at System.Net.Http.HttpConnection.SendAsyncCore(HttpRequestMessage request, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at System.Net.Http.HttpConnection.SendAsyncCore(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithNtConnectionAuthAsync(HttpConnection connection, HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.DiagnosticsHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.FinishSendAsyncUnbuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   --- End of inner exception stack trace ---
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)StackTrace -    at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)Message - Failed to proxy the request to http://localhost:64809/polyfills.js, because the request to the proxy target failed. Check that the proxy target server is running and accepting requests to http://localhost:64809/.

The underlying exception message was 'An error occurred while sending the request.'.Check the InnerException for more details.Source - Microsoft.AspNetCore.SpaServices.ExtensionsInnerException - System.Net.Http.HttpRequestException: An error occurred while sending the request.
 ---> System.IO.IOException: Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host..
 ---> System.Net.Sockets.SocketException (10054): An existing connection was forcibly closed by the remote host.
   --- End of inner exception stack trace ---
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.GetResult(Int16 token)
   at System.Net.Http.HttpConnection.FillAsync()
   at System.Net.Http.HttpConnection.ReadNextResponseHeaderLineAsync(Boolean foldedHeadersAllowed)
   at System.Net.Http.HttpConnection.SendAsyncCore(HttpRequestMessage request, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at System.Net.Http.HttpConnection.SendAsyncCore(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithNtConnectionAuthAsync(HttpConnection connection, HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.DiagnosticsHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.FinishSendAsyncUnbuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 12:06:34 +03:00 | Level - Error |
----------------------------------------------------------------------
EventId : { Id: 1, Name: "UnhandledException" }
SourceContext : "Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware"
RequestId : "0HM84IQTJN23E:0000016D"
RequestPath : "/main.js"
SpanId : "|8954f94f-4b6d7db69a6ff0c8."
TraceId : "8954f94f-4b6d7db69a6ff0c8"
ParentId : ""
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
----------------------EXCEPTION DETAILS-------------------------------
Exception - System.Net.Http.HttpRequestException: Failed to proxy the request to http://localhost:64809/main.js, because the request to the proxy target failed. Check that the proxy target server is running and accepting requests to http://localhost:64809/.

The underlying exception message was 'An error occurred while sending the request.'.Check the InnerException for more details.
 ---> System.Net.Http.HttpRequestException: An error occurred while sending the request.
 ---> System.IO.IOException: Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host..
 ---> System.Net.Sockets.SocketException (10054): An existing connection was forcibly closed by the remote host.
   --- End of inner exception stack trace ---
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.GetResult(Int16 token)
   at System.Net.Http.HttpConnection.FillAsync()
   at System.Net.Http.HttpConnection.ReadNextResponseHeaderLineAsync(Boolean foldedHeadersAllowed)
   at System.Net.Http.HttpConnection.SendAsyncCore(HttpRequestMessage request, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at System.Net.Http.HttpConnection.SendAsyncCore(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithNtConnectionAuthAsync(HttpConnection connection, HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.DiagnosticsHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.FinishSendAsyncUnbuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   --- End of inner exception stack trace ---
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)StackTrace -    at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)Message - Failed to proxy the request to http://localhost:64809/main.js, because the request to the proxy target failed. Check that the proxy target server is running and accepting requests to http://localhost:64809/.

The underlying exception message was 'An error occurred while sending the request.'.Check the InnerException for more details.Source - Microsoft.AspNetCore.SpaServices.ExtensionsInnerException - System.Net.Http.HttpRequestException: An error occurred while sending the request.
 ---> System.IO.IOException: Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host..
 ---> System.Net.Sockets.SocketException (10054): An existing connection was forcibly closed by the remote host.
   --- End of inner exception stack trace ---
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.GetResult(Int16 token)
   at System.Net.Http.HttpConnection.FillAsync()
   at System.Net.Http.HttpConnection.ReadNextResponseHeaderLineAsync(Boolean foldedHeadersAllowed)
   at System.Net.Http.HttpConnection.SendAsyncCore(HttpRequestMessage request, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at System.Net.Http.HttpConnection.SendAsyncCore(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithNtConnectionAuthAsync(HttpConnection connection, HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.DiagnosticsHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.FinishSendAsyncUnbuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 12:06:34 +03:00 | Level - Error |
----------------------------------------------------------------------
EventId : { Id: 1, Name: "UnhandledException" }
SourceContext : "Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware"
RequestId : "0HM84IQTJN23E:00000167"
RequestPath : "/styles.js"
SpanId : "|8954f94c-4b6d7db69a6ff0c8."
TraceId : "8954f94c-4b6d7db69a6ff0c8"
ParentId : ""
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
----------------------EXCEPTION DETAILS-------------------------------
Exception - System.Net.Http.HttpRequestException: Failed to proxy the request to http://localhost:64809/styles.js, because the request to the proxy target failed. Check that the proxy target server is running and accepting requests to http://localhost:64809/.

The underlying exception message was 'An error occurred while sending the request.'.Check the InnerException for more details.
 ---> System.Net.Http.HttpRequestException: An error occurred while sending the request.
 ---> System.IO.IOException: Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host..
 ---> System.Net.Sockets.SocketException (10054): An existing connection was forcibly closed by the remote host.
   --- End of inner exception stack trace ---
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.GetResult(Int16 token)
   at System.Net.Http.HttpConnection.FillAsync()
   at System.Net.Http.HttpConnection.ReadNextResponseHeaderLineAsync(Boolean foldedHeadersAllowed)
   at System.Net.Http.HttpConnection.SendAsyncCore(HttpRequestMessage request, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at System.Net.Http.HttpConnection.SendAsyncCore(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithNtConnectionAuthAsync(HttpConnection connection, HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.DiagnosticsHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.FinishSendAsyncUnbuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   --- End of inner exception stack trace ---
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)StackTrace -    at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)Message - Failed to proxy the request to http://localhost:64809/styles.js, because the request to the proxy target failed. Check that the proxy target server is running and accepting requests to http://localhost:64809/.

The underlying exception message was 'An error occurred while sending the request.'.Check the InnerException for more details.Source - Microsoft.AspNetCore.SpaServices.ExtensionsInnerException - System.Net.Http.HttpRequestException: An error occurred while sending the request.
 ---> System.IO.IOException: Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host..
 ---> System.Net.Sockets.SocketException (10054): An existing connection was forcibly closed by the remote host.
   --- End of inner exception stack trace ---
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.GetResult(Int16 token)
   at System.Net.Http.HttpConnection.FillAsync()
   at System.Net.Http.HttpConnection.ReadNextResponseHeaderLineAsync(Boolean foldedHeadersAllowed)
   at System.Net.Http.HttpConnection.SendAsyncCore(HttpRequestMessage request, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at System.Net.Http.HttpConnection.SendAsyncCore(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithNtConnectionAuthAsync(HttpConnection connection, HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.DiagnosticsHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.FinishSendAsyncUnbuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 12:06:34 +03:00 | Level - Error |
----------------------------------------------------------------------
EventId : { Id: 1, Name: "UnhandledException" }
SourceContext : "Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware"
RequestId : "0HM84IQTJN23E:0000016B"
RequestPath : "/vendor.js"
SpanId : "|8954f94e-4b6d7db69a6ff0c8."
TraceId : "8954f94e-4b6d7db69a6ff0c8"
ParentId : ""
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
----------------------EXCEPTION DETAILS-------------------------------
Exception - System.Net.Http.HttpRequestException: Failed to proxy the request to http://localhost:64809/vendor.js, because the request to the proxy target failed. Check that the proxy target server is running and accepting requests to http://localhost:64809/.

The underlying exception message was 'An error occurred while sending the request.'.Check the InnerException for more details.
 ---> System.Net.Http.HttpRequestException: An error occurred while sending the request.
 ---> System.IO.IOException: Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host..
 ---> System.Net.Sockets.SocketException (10054): An existing connection was forcibly closed by the remote host.
   --- End of inner exception stack trace ---
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.GetResult(Int16 token)
   at System.Net.Http.HttpConnection.FillAsync()
   at System.Net.Http.HttpConnection.ReadNextResponseHeaderLineAsync(Boolean foldedHeadersAllowed)
   at System.Net.Http.HttpConnection.SendAsyncCore(HttpRequestMessage request, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at System.Net.Http.HttpConnection.SendAsyncCore(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithNtConnectionAuthAsync(HttpConnection connection, HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.DiagnosticsHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.FinishSendAsyncUnbuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   --- End of inner exception stack trace ---
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)StackTrace -    at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)Message - Failed to proxy the request to http://localhost:64809/vendor.js, because the request to the proxy target failed. Check that the proxy target server is running and accepting requests to http://localhost:64809/.

The underlying exception message was 'An error occurred while sending the request.'.Check the InnerException for more details.Source - Microsoft.AspNetCore.SpaServices.ExtensionsInnerException - System.Net.Http.HttpRequestException: An error occurred while sending the request.
 ---> System.IO.IOException: Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host..
 ---> System.Net.Sockets.SocketException (10054): An existing connection was forcibly closed by the remote host.
   --- End of inner exception stack trace ---
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.GetResult(Int16 token)
   at System.Net.Http.HttpConnection.FillAsync()
   at System.Net.Http.HttpConnection.ReadNextResponseHeaderLineAsync(Boolean foldedHeadersAllowed)
   at System.Net.Http.HttpConnection.SendAsyncCore(HttpRequestMessage request, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at System.Net.Http.HttpConnection.SendAsyncCore(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithNtConnectionAuthAsync(HttpConnection connection, HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.DiagnosticsHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.FinishSendAsyncUnbuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 12:06:34 +03:00 | Level - Error |
----------------------------------------------------------------------
EventId : { Id: 1, Name: "UnhandledException" }
SourceContext : "Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware"
RequestId : "0HM84IQTJN23E:00000169"
RequestPath : "/scripts.js"
SpanId : "|8954f94d-4b6d7db69a6ff0c8."
TraceId : "8954f94d-4b6d7db69a6ff0c8"
ParentId : ""
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
----------------------EXCEPTION DETAILS-------------------------------
Exception - System.Net.Http.HttpRequestException: Failed to proxy the request to http://localhost:64809/scripts.js, because the request to the proxy target failed. Check that the proxy target server is running and accepting requests to http://localhost:64809/.

The underlying exception message was 'An error occurred while sending the request.'.Check the InnerException for more details.
 ---> System.Net.Http.HttpRequestException: An error occurred while sending the request.
 ---> System.IO.IOException: Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host..
 ---> System.Net.Sockets.SocketException (10054): An existing connection was forcibly closed by the remote host.
   --- End of inner exception stack trace ---
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.GetResult(Int16 token)
   at System.Net.Http.HttpConnection.FillAsync()
   at System.Net.Http.HttpConnection.ReadNextResponseHeaderLineAsync(Boolean foldedHeadersAllowed)
   at System.Net.Http.HttpConnection.SendAsyncCore(HttpRequestMessage request, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at System.Net.Http.HttpConnection.SendAsyncCore(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithNtConnectionAuthAsync(HttpConnection connection, HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.DiagnosticsHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.FinishSendAsyncUnbuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   --- End of inner exception stack trace ---
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)StackTrace -    at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)Message - Failed to proxy the request to http://localhost:64809/scripts.js, because the request to the proxy target failed. Check that the proxy target server is running and accepting requests to http://localhost:64809/.

The underlying exception message was 'An error occurred while sending the request.'.Check the InnerException for more details.Source - Microsoft.AspNetCore.SpaServices.ExtensionsInnerException - System.Net.Http.HttpRequestException: An error occurred while sending the request.
 ---> System.IO.IOException: Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host..
 ---> System.Net.Sockets.SocketException (10054): An existing connection was forcibly closed by the remote host.
   --- End of inner exception stack trace ---
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.GetResult(Int16 token)
   at System.Net.Http.HttpConnection.FillAsync()
   at System.Net.Http.HttpConnection.ReadNextResponseHeaderLineAsync(Boolean foldedHeadersAllowed)
   at System.Net.Http.HttpConnection.SendAsyncCore(HttpRequestMessage request, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at System.Net.Http.HttpConnection.SendAsyncCore(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithNtConnectionAuthAsync(HttpConnection connection, HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.DiagnosticsHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.FinishSendAsyncUnbuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 12:06:35 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 12:06:35 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 12:06:35 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 12:06:35 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 12:06:39 +03:00 | Level - Error |
----------------------------------------------------------------------
EventId : { Id: 1, Name: "UnhandledException" }
SourceContext : "Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware"
RequestId : "0HM84IQTJN23E:00000163"
RequestPath : "/runtime.js"
SpanId : "|8954f94a-4b6d7db69a6ff0c8."
TraceId : "8954f94a-4b6d7db69a6ff0c8"
ParentId : ""
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
----------------------EXCEPTION DETAILS-------------------------------
Exception - System.Net.Http.HttpRequestException: Failed to proxy the request to http://localhost:64809/runtime.js, because the request to the proxy target failed. Check that the proxy target server is running and accepting requests to http://localhost:64809/.

The underlying exception message was 'No connection could be made because the target machine actively refused it.'.Check the InnerException for more details.
 ---> System.Net.Http.HttpRequestException: No connection could be made because the target machine actively refused it.
 ---> System.Net.Sockets.SocketException (10061): No connection could be made because the target machine actively refused it.
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.ConnectAsync(HttpRequestMessage request, Boolean allowHttp2, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.CreateHttp11ConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.GetHttpConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.DiagnosticsHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.FinishSendAsyncUnbuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   --- End of inner exception stack trace ---
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)StackTrace -    at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)Message - Failed to proxy the request to http://localhost:64809/runtime.js, because the request to the proxy target failed. Check that the proxy target server is running and accepting requests to http://localhost:64809/.

The underlying exception message was 'No connection could be made because the target machine actively refused it.'.Check the InnerException for more details.Source - Microsoft.AspNetCore.SpaServices.ExtensionsInnerException - System.Net.Http.HttpRequestException: No connection could be made because the target machine actively refused it.
 ---> System.Net.Sockets.SocketException (10061): No connection could be made because the target machine actively refused it.
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.ConnectAsync(HttpRequestMessage request, Boolean allowHttp2, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.CreateHttp11ConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.GetHttpConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.DiagnosticsHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.FinishSendAsyncUnbuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 12:06:59 +03:00 | Level - Error |
----------------------------------------------------------------------
EventId : { Id: 1, Name: "UnhandledException" }
SourceContext : "Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware"
RequestId : "0HM84IQTJN23E:00000171"
RequestPath : "/"
SpanId : "|8954f951-4b6d7db69a6ff0c8."
TraceId : "8954f951-4b6d7db69a6ff0c8"
ParentId : ""
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
----------------------EXCEPTION DETAILS-------------------------------
Exception - System.Net.Http.HttpRequestException: Failed to proxy the request to http://localhost:64809/, because the request to the proxy target failed. Check that the proxy target server is running and accepting requests to http://localhost:64809/.

The underlying exception message was 'No connection could be made because the target machine actively refused it.'.Check the InnerException for more details.
 ---> System.Net.Http.HttpRequestException: No connection could be made because the target machine actively refused it.
 ---> System.Net.Sockets.SocketException (10061): No connection could be made because the target machine actively refused it.
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.ConnectAsync(HttpRequestMessage request, Boolean allowHttp2, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.CreateHttp11ConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.GetHttpConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.DiagnosticsHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.FinishSendAsyncUnbuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   --- End of inner exception stack trace ---
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)StackTrace -    at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)Message - Failed to proxy the request to http://localhost:64809/, because the request to the proxy target failed. Check that the proxy target server is running and accepting requests to http://localhost:64809/.

The underlying exception message was 'No connection could be made because the target machine actively refused it.'.Check the InnerException for more details.Source - Microsoft.AspNetCore.SpaServices.ExtensionsInnerException - System.Net.Http.HttpRequestException: No connection could be made because the target machine actively refused it.
 ---> System.Net.Sockets.SocketException (10061): No connection could be made because the target machine actively refused it.
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.ConnectAsync(HttpRequestMessage request, Boolean allowHttp2, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.CreateHttp11ConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.GetHttpConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.DiagnosticsHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.FinishSendAsyncUnbuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 12:07:33 +03:00 | Level - Error |
----------------------------------------------------------------------
EventId : { Id: 1, Name: "UnhandledException" }
SourceContext : "Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware"
RequestId : "0HM84IQTJN23E:00000173"
RequestPath : "/"
SpanId : "|8954f952-4b6d7db69a6ff0c8."
TraceId : "8954f952-4b6d7db69a6ff0c8"
ParentId : ""
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 16052
ProcessName : "CMS_CORE_NG"
----------------------EXCEPTION DETAILS-------------------------------
Exception - System.Net.Http.HttpRequestException: Failed to proxy the request to http://localhost:64809/, because the request to the proxy target failed. Check that the proxy target server is running and accepting requests to http://localhost:64809/.

The underlying exception message was 'No connection could be made because the target machine actively refused it.'.Check the InnerException for more details.
 ---> System.Net.Http.HttpRequestException: No connection could be made because the target machine actively refused it.
 ---> System.Net.Sockets.SocketException (10061): No connection could be made because the target machine actively refused it.
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.ConnectAsync(HttpRequestMessage request, Boolean allowHttp2, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.CreateHttp11ConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.GetHttpConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.DiagnosticsHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.FinishSendAsyncUnbuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   --- End of inner exception stack trace ---
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)StackTrace -    at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)Message - Failed to proxy the request to http://localhost:64809/, because the request to the proxy target failed. Check that the proxy target server is running and accepting requests to http://localhost:64809/.

The underlying exception message was 'No connection could be made because the target machine actively refused it.'.Check the InnerException for more details.Source - Microsoft.AspNetCore.SpaServices.ExtensionsInnerException - System.Net.Http.HttpRequestException: No connection could be made because the target machine actively refused it.
 ---> System.Net.Sockets.SocketException (10061): No connection could be made because the target machine actively refused it.
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.ConnectAsync(HttpRequestMessage request, Boolean allowHttp2, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.CreateHttp11ConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.GetHttpConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.DiagnosticsHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.FinishSendAsyncUnbuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 12:08:17 +03:00 | Level - Error |
----------------------------------------------------------------------
Error : "One or more errors occurred. (A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server))"
StackTrace : "   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at System.Threading.Tasks.Task.Wait()
   at CMS_CORE_NG.Program.Main(String[] args) in C:\TFS\identity windo\CMS_CORE_NG\CMS_CORE_NG\Program.cs:line 36"
InnerException : "Microsoft.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)
 ---> System.ComponentModel.Win32Exception (2): The system cannot find the file specified.
   at Microsoft.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, Boolean applyTransientFaultHandling, String accessToken, DbConnectionPool pool, SqlAuthenticationProviderManager sqlAuthProviderManager)
   at Microsoft.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnectionAsync(Boolean errorsExpected, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnectionAsync(Boolean errorsExpected, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenAsync(CancellationToken cancellationToken, Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabaseCreator.EnsureCreatedAsync(CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabaseCreator.EnsureCreatedAsync(CancellationToken cancellationToken)
   at DataService.DbContextInitializer.Initialize(DataProtectionKeysContext dataProtectionKeysContext, ApplicationDbContext applicationDbContext, IFunctionalSvc functionalSvc, ICountrySvc countrySvc) in C:\TFS\identity windo\CMS_CORE_NG\DataService\DbContextInitializer.cs:line 15
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:2,State:0,Class:20"
Source : "System.Private.CoreLib"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 2640
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 12:08:17 +03:00 | Level - Error |
----------------------------------------------------------------------
database : "DataProtectionKeysContext"
server : "."
EventId : { Id: 20004, Name: "Microsoft.EntityFrameworkCore.Database.Connection.ConnectionError" }
SourceContext : "Microsoft.EntityFrameworkCore.Database.Connection"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 2640
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 12:08:17 +03:00 | Level - Error |
----------------------------------------------------------------------
contextType : "DataService.DataProtectionKeysContext"
newline : "
"
error : "Microsoft.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)
 ---> System.ComponentModel.Win32Exception (2): The system cannot find the file specified.
   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at Microsoft.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at Microsoft.Data.SqlClient.SqlConnection.Open()
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnection(Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.Open(Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReader(RelationalCommandParameterObject parameterObject)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.Enumerator.InitializeReader(DbContext _, Boolean result)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.Execute[TState,TResult](TState state, Func`3 operation, Func`3 verifySucceeded)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.Enumerator.MoveNext()
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:2,State:0,Class:20"
EventId : { Id: 10100, Name: "Microsoft.EntityFrameworkCore.Query.QueryIterationFailed" }
SourceContext : "Microsoft.EntityFrameworkCore.Query"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 2640
ProcessName : "CMS_CORE_NG"
----------------------EXCEPTION DETAILS-------------------------------
Exception - Microsoft.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)
 ---> System.ComponentModel.Win32Exception (2): The system cannot find the file specified.
   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at Microsoft.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at Microsoft.Data.SqlClient.SqlConnection.Open()
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnection(Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.Open(Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReader(RelationalCommandParameterObject parameterObject)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.Enumerator.InitializeReader(DbContext _, Boolean result)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.Execute[TState,TResult](TState state, Func`3 operation, Func`3 verifySucceeded)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.Enumerator.MoveNext()
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:2,State:0,Class:20StackTrace -    at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at Microsoft.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at Microsoft.Data.SqlClient.SqlConnection.Open()
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnection(Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.Open(Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReader(RelationalCommandParameterObject parameterObject)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.Enumerator.InitializeReader(DbContext _, Boolean result)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.Execute[TState,TResult](TState state, Func`3 operation, Func`3 verifySucceeded)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.Enumerator.MoveNext()Message - A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)Source - Core Microsoft SqlClient Data ProviderInnerException - System.ComponentModel.Win32Exception (2): The system cannot find the file specified.---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 12:08:17 +03:00 | Level - Error |
----------------------------------------------------------------------
EventId : { Id: 48 }
SourceContext : "Microsoft.AspNetCore.DataProtection.KeyManagement.KeyRingProvider"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 2640
ProcessName : "CMS_CORE_NG"
----------------------EXCEPTION DETAILS-------------------------------
Exception - Microsoft.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)
 ---> System.ComponentModel.Win32Exception (2): The system cannot find the file specified.
   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at Microsoft.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at Microsoft.Data.SqlClient.SqlConnection.Open()
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnection(Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.Open(Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReader(RelationalCommandParameterObject parameterObject)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.Enumerator.InitializeReader(DbContext _, Boolean result)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.Execute[TState,TResult](TState state, Func`3 operation, Func`3 verifySucceeded)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.Enumerator.MoveNext()
   at System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   at Microsoft.AspNetCore.DataProtection.EntityFrameworkCore.EntityFrameworkCoreXmlRepository`1.GetAllElements()
   at Microsoft.AspNetCore.DataProtection.KeyManagement.XmlKeyManager.GetAllKeys()
   at Microsoft.AspNetCore.DataProtection.KeyManagement.KeyRingProvider.CreateCacheableKeyRingCore(DateTimeOffset now, IKey keyJustAdded)
   at Microsoft.AspNetCore.DataProtection.KeyManagement.KeyRingProvider.Microsoft.AspNetCore.DataProtection.KeyManagement.Internal.ICacheableKeyRingProvider.GetCacheableKeyRing(DateTimeOffset now)
   at Microsoft.AspNetCore.DataProtection.KeyManagement.KeyRingProvider.GetCurrentKeyRingCore(DateTime utcNow, Boolean forceRefresh)
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:2,State:0,Class:20StackTrace -    at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at Microsoft.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at Microsoft.Data.SqlClient.SqlConnection.Open()
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnection(Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.Open(Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReader(RelationalCommandParameterObject parameterObject)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.Enumerator.InitializeReader(DbContext _, Boolean result)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.Execute[TState,TResult](TState state, Func`3 operation, Func`3 verifySucceeded)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.Enumerator.MoveNext()
   at System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   at Microsoft.AspNetCore.DataProtection.EntityFrameworkCore.EntityFrameworkCoreXmlRepository`1.GetAllElements()
   at Microsoft.AspNetCore.DataProtection.KeyManagement.XmlKeyManager.GetAllKeys()
   at Microsoft.AspNetCore.DataProtection.KeyManagement.KeyRingProvider.CreateCacheableKeyRingCore(DateTimeOffset now, IKey keyJustAdded)
   at Microsoft.AspNetCore.DataProtection.KeyManagement.KeyRingProvider.Microsoft.AspNetCore.DataProtection.KeyManagement.Internal.ICacheableKeyRingProvider.GetCacheableKeyRing(DateTimeOffset now)
   at Microsoft.AspNetCore.DataProtection.KeyManagement.KeyRingProvider.GetCurrentKeyRingCore(DateTime utcNow, Boolean forceRefresh)Message - A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)Source - Core Microsoft SqlClient Data ProviderInnerException - System.ComponentModel.Win32Exception (2): The system cannot find the file specified.---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 12:08:19 +03:00 | Level - Error |
----------------------------------------------------------------------
database : "DataProtectionKeysContext"
server : "."
EventId : { Id: 20004, Name: "Microsoft.EntityFrameworkCore.Database.Connection.ConnectionError" }
SourceContext : "Microsoft.EntityFrameworkCore.Database.Connection"
RequestId : "0HM84KK0GCHVR:00000001"
RequestPath : "/"
SpanId : "|9c56d964-499a7dafbc3cfb6d."
TraceId : "9c56d964-499a7dafbc3cfb6d"
ParentId : ""
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 2640
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 12:08:19 +03:00 | Level - Error |
----------------------------------------------------------------------
contextType : "DataService.DataProtectionKeysContext"
newline : "
"
error : "Microsoft.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)
 ---> System.ComponentModel.Win32Exception (2): The system cannot find the file specified.
   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at Microsoft.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at Microsoft.Data.SqlClient.SqlConnection.Open()
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnection(Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.Open(Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReader(RelationalCommandParameterObject parameterObject)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.Enumerator.InitializeReader(DbContext _, Boolean result)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.Execute[TState,TResult](TState state, Func`3 operation, Func`3 verifySucceeded)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.Enumerator.MoveNext()
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:2,State:0,Class:20"
EventId : { Id: 10100, Name: "Microsoft.EntityFrameworkCore.Query.QueryIterationFailed" }
SourceContext : "Microsoft.EntityFrameworkCore.Query"
RequestId : "0HM84KK0GCHVR:00000001"
RequestPath : "/"
SpanId : "|9c56d964-499a7dafbc3cfb6d."
TraceId : "9c56d964-499a7dafbc3cfb6d"
ParentId : ""
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 2640
ProcessName : "CMS_CORE_NG"
----------------------EXCEPTION DETAILS-------------------------------
Exception - Microsoft.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)
 ---> System.ComponentModel.Win32Exception (2): The system cannot find the file specified.
   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at Microsoft.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at Microsoft.Data.SqlClient.SqlConnection.Open()
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnection(Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.Open(Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReader(RelationalCommandParameterObject parameterObject)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.Enumerator.InitializeReader(DbContext _, Boolean result)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.Execute[TState,TResult](TState state, Func`3 operation, Func`3 verifySucceeded)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.Enumerator.MoveNext()
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:2,State:0,Class:20StackTrace -    at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at Microsoft.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at Microsoft.Data.SqlClient.SqlConnection.Open()
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnection(Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.Open(Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReader(RelationalCommandParameterObject parameterObject)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.Enumerator.InitializeReader(DbContext _, Boolean result)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.Execute[TState,TResult](TState state, Func`3 operation, Func`3 verifySucceeded)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.Enumerator.MoveNext()Message - A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)Source - Core Microsoft SqlClient Data ProviderInnerException - System.ComponentModel.Win32Exception (2): The system cannot find the file specified.---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 12:08:19 +03:00 | Level - Error |
----------------------------------------------------------------------
EventId : { Id: 48 }
SourceContext : "Microsoft.AspNetCore.DataProtection.KeyManagement.KeyRingProvider"
RequestId : "0HM84KK0GCHVR:00000001"
RequestPath : "/"
SpanId : "|9c56d964-499a7dafbc3cfb6d."
TraceId : "9c56d964-499a7dafbc3cfb6d"
ParentId : ""
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 2640
ProcessName : "CMS_CORE_NG"
----------------------EXCEPTION DETAILS-------------------------------
Exception - Microsoft.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)
 ---> System.ComponentModel.Win32Exception (2): The system cannot find the file specified.
   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at Microsoft.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at Microsoft.Data.SqlClient.SqlConnection.Open()
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnection(Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.Open(Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReader(RelationalCommandParameterObject parameterObject)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.Enumerator.InitializeReader(DbContext _, Boolean result)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.Execute[TState,TResult](TState state, Func`3 operation, Func`3 verifySucceeded)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.Enumerator.MoveNext()
   at System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   at Microsoft.AspNetCore.DataProtection.EntityFrameworkCore.EntityFrameworkCoreXmlRepository`1.GetAllElements()
   at Microsoft.AspNetCore.DataProtection.KeyManagement.XmlKeyManager.GetAllKeys()
   at Microsoft.AspNetCore.DataProtection.KeyManagement.KeyRingProvider.CreateCacheableKeyRingCore(DateTimeOffset now, IKey keyJustAdded)
   at Microsoft.AspNetCore.DataProtection.KeyManagement.KeyRingProvider.Microsoft.AspNetCore.DataProtection.KeyManagement.Internal.ICacheableKeyRingProvider.GetCacheableKeyRing(DateTimeOffset now)
   at Microsoft.AspNetCore.DataProtection.KeyManagement.KeyRingProvider.GetCurrentKeyRingCore(DateTime utcNow, Boolean forceRefresh)
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:2,State:0,Class:20StackTrace -    at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at Microsoft.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at Microsoft.Data.SqlClient.SqlConnection.Open()
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnection(Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.Open(Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReader(RelationalCommandParameterObject parameterObject)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.Enumerator.InitializeReader(DbContext _, Boolean result)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.Execute[TState,TResult](TState state, Func`3 operation, Func`3 verifySucceeded)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.Enumerator.MoveNext()
   at System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   at Microsoft.AspNetCore.DataProtection.EntityFrameworkCore.EntityFrameworkCoreXmlRepository`1.GetAllElements()
   at Microsoft.AspNetCore.DataProtection.KeyManagement.XmlKeyManager.GetAllKeys()
   at Microsoft.AspNetCore.DataProtection.KeyManagement.KeyRingProvider.CreateCacheableKeyRingCore(DateTimeOffset now, IKey keyJustAdded)
   at Microsoft.AspNetCore.DataProtection.KeyManagement.KeyRingProvider.Microsoft.AspNetCore.DataProtection.KeyManagement.Internal.ICacheableKeyRingProvider.GetCacheableKeyRing(DateTimeOffset now)
   at Microsoft.AspNetCore.DataProtection.KeyManagement.KeyRingProvider.GetCurrentKeyRingCore(DateTime utcNow, Boolean forceRefresh)Message - A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)Source - Core Microsoft SqlClient Data ProviderInnerException - System.ComponentModel.Win32Exception (2): The system cannot find the file specified.---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 12:10:25 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 2640
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 12:10:25 +03:00 | Level - Error |
----------------------------------------------------------------------
EventId : { Id: 1, Name: "UnhandledException" }
SourceContext : "Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware"
RequestId : "0HM84KK0GCHVR:00000001"
RequestPath : "/"
SpanId : "|9c56d964-499a7dafbc3cfb6d."
TraceId : "9c56d964-499a7dafbc3cfb6d"
ParentId : ""
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 2640
ProcessName : "CMS_CORE_NG"
----------------------EXCEPTION DETAILS-------------------------------
Exception - System.Security.Cryptography.CryptographicException: An error occurred while trying to encrypt the provided data. Refer to the inner exception for more information.
 ---> Microsoft.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)
 ---> System.ComponentModel.Win32Exception (2): The system cannot find the file specified.
   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at Microsoft.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at Microsoft.Data.SqlClient.SqlConnection.Open()
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnection(Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.Open(Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReader(RelationalCommandParameterObject parameterObject)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.Enumerator.InitializeReader(DbContext _, Boolean result)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.Execute[TState,TResult](TState state, Func`3 operation, Func`3 verifySucceeded)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.Enumerator.MoveNext()
   at System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   at Microsoft.AspNetCore.DataProtection.EntityFrameworkCore.EntityFrameworkCoreXmlRepository`1.GetAllElements()
   at Microsoft.AspNetCore.DataProtection.KeyManagement.XmlKeyManager.GetAllKeys()
   at Microsoft.AspNetCore.DataProtection.KeyManagement.KeyRingProvider.CreateCacheableKeyRingCore(DateTimeOffset now, IKey keyJustAdded)
   at Microsoft.AspNetCore.DataProtection.KeyManagement.KeyRingProvider.Microsoft.AspNetCore.DataProtection.KeyManagement.Internal.ICacheableKeyRingProvider.GetCacheableKeyRing(DateTimeOffset now)
   at Microsoft.AspNetCore.DataProtection.KeyManagement.KeyRingProvider.GetCurrentKeyRingCore(DateTime utcNow, Boolean forceRefresh)
   at Microsoft.AspNetCore.DataProtection.KeyManagement.KeyRingProvider.GetCurrentKeyRing()
   at Microsoft.AspNetCore.DataProtection.KeyManagement.KeyRingBasedDataProtector.Protect(Byte[] plaintext)
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:2,State:0,Class:20
   --- End of inner exception stack trace ---
   at Microsoft.AspNetCore.DataProtection.KeyManagement.KeyRingBasedDataProtector.Protect(Byte[] plaintext)
   at Microsoft.AspNetCore.Antiforgery.DefaultAntiforgeryTokenSerializer.Serialize(AntiforgeryToken token)
   at Microsoft.AspNetCore.Antiforgery.DefaultAntiforgery.Serialize(IAntiforgeryFeature antiforgeryFeature)
   at Microsoft.AspNetCore.Antiforgery.DefaultAntiforgery.GetAndStoreTokens(HttpContext httpContext)
   at CMS_CORE_NG.Startup.<>c__DisplayClass5_1.<Configure>b__3(HttpContext context) in C:\TFS\identity windo\CMS_CORE_NG\CMS_CORE_NG\Startup.cs:line 261
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)StackTrace -    at Microsoft.AspNetCore.DataProtection.KeyManagement.KeyRingBasedDataProtector.Protect(Byte[] plaintext)
   at Microsoft.AspNetCore.Antiforgery.DefaultAntiforgeryTokenSerializer.Serialize(AntiforgeryToken token)
   at Microsoft.AspNetCore.Antiforgery.DefaultAntiforgery.Serialize(IAntiforgeryFeature antiforgeryFeature)
   at Microsoft.AspNetCore.Antiforgery.DefaultAntiforgery.GetAndStoreTokens(HttpContext httpContext)
   at CMS_CORE_NG.Startup.<>c__DisplayClass5_1.<Configure>b__3(HttpContext context) in C:\TFS\identity windo\CMS_CORE_NG\CMS_CORE_NG\Startup.cs:line 261
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)Message - An error occurred while trying to encrypt the provided data. Refer to the inner exception for more information.Source - Microsoft.AspNetCore.DataProtectionInnerException - Microsoft.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)
 ---> System.ComponentModel.Win32Exception (2): The system cannot find the file specified.
   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at Microsoft.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at Microsoft.Data.SqlClient.SqlConnection.Open()
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnection(Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.Open(Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReader(RelationalCommandParameterObject parameterObject)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.Enumerator.InitializeReader(DbContext _, Boolean result)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.Execute[TState,TResult](TState state, Func`3 operation, Func`3 verifySucceeded)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.Enumerator.MoveNext()
   at System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   at Microsoft.AspNetCore.DataProtection.EntityFrameworkCore.EntityFrameworkCoreXmlRepository`1.GetAllElements()
   at Microsoft.AspNetCore.DataProtection.KeyManagement.XmlKeyManager.GetAllKeys()
   at Microsoft.AspNetCore.DataProtection.KeyManagement.KeyRingProvider.CreateCacheableKeyRingCore(DateTimeOffset now, IKey keyJustAdded)
   at Microsoft.AspNetCore.DataProtection.KeyManagement.KeyRingProvider.Microsoft.AspNetCore.DataProtection.KeyManagement.Internal.ICacheableKeyRingProvider.GetCacheableKeyRing(DateTimeOffset now)
   at Microsoft.AspNetCore.DataProtection.KeyManagement.KeyRingProvider.GetCurrentKeyRingCore(DateTime utcNow, Boolean forceRefresh)
   at Microsoft.AspNetCore.DataProtection.KeyManagement.KeyRingProvider.GetCurrentKeyRing()
   at Microsoft.AspNetCore.DataProtection.KeyManagement.KeyRingBasedDataProtector.Protect(Byte[] plaintext)
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:2,State:0,Class:20---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 12:11:22 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-T70OQEC"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 6.2.9200.0"
Version : "3.1.14"
UserName : "hedva"
ProcessId : 4944
ProcessName : "CMS_CORE_NG"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 18:43:42 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-4GF5G0T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19041.0"
Version : "3.1.11"
UserName : "ruti"
ProcessId : 24456
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 18:43:42 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-4GF5G0T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19041.0"
Version : "3.1.11"
UserName : "ruti"
ProcessId : 24456
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 18:43:43 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-4GF5G0T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19041.0"
Version : "3.1.11"
UserName : "ruti"
ProcessId : 24456
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 18:43:49 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-4GF5G0T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19041.0"
Version : "3.1.11"
UserName : "ruti"
ProcessId : 24456
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 18:43:49 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-4GF5G0T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19041.0"
Version : "3.1.11"
UserName : "ruti"
ProcessId : 24456
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 18:43:52 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-4GF5G0T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19041.0"
Version : "3.1.11"
UserName : "ruti"
ProcessId : 24456
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 18:43:52 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-4GF5G0T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19041.0"
Version : "3.1.11"
UserName : "ruti"
ProcessId : 24456
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 18:43:52 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-4GF5G0T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19041.0"
Version : "3.1.11"
UserName : "ruti"
ProcessId : 24456
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 18:43:53 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-4GF5G0T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19041.0"
Version : "3.1.11"
UserName : "ruti"
ProcessId : 24456
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 18:43:58 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-4GF5G0T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19041.0"
Version : "3.1.11"
UserName : "ruti"
ProcessId : 24456
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 18:43:58 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-4GF5G0T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19041.0"
Version : "3.1.11"
UserName : "ruti"
ProcessId : 24456
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 18:43:59 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-4GF5G0T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19041.0"
Version : "3.1.11"
UserName : "ruti"
ProcessId : 24456
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 18:44:02 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-4GF5G0T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19041.0"
Version : "3.1.11"
UserName : "ruti"
ProcessId : 24456
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 18:44:03 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-4GF5G0T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19041.0"
Version : "3.1.11"
UserName : "ruti"
ProcessId : 24456
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 18:44:06 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-4GF5G0T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19041.0"
Version : "3.1.11"
UserName : "ruti"
ProcessId : 24456
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 18:44:08 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-4GF5G0T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19041.0"
Version : "3.1.11"
UserName : "ruti"
ProcessId : 24456
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 18:44:26 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-4GF5G0T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19041.0"
Version : "3.1.11"
UserName : "ruti"
ProcessId : 24456
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 18:44:47 +03:00 | Level - Error |
----------------------------------------------------------------------
RequestId : "80000057-0003-fd00-b63f-84710c7967bb"
RequestPath : "/admin"
SpanId : "|1f54dde1-49ca3057093d0966."
TraceId : "1f54dde1-49ca3057093d0966"
ParentId : ""
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-4GF5G0T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19041.0"
Version : "3.1.11"
UserName : "ruti"
ProcessId : 24456
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 18:46:49 +03:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-4GF5G0T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19041.0"
Version : "3.1.11"
UserName : "ruti"
ProcessId : 10880
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 18:49:14 +03:00 | Level - Error |
----------------------------------------------------------------------
RequestId : "8000008b-0000-fd00-b63f-84710c7967bb"
RequestPath : "/admin"
SpanId : "|43837315-45e77c26e6fd7ea9."
TraceId : "43837315-45e77c26e6fd7ea9"
ParentId : ""
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-4GF5G0T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19041.0"
Version : "3.1.11"
UserName : "ruti"
ProcessId : 10880
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 18:55:59 +03:00 | Level - Error |
----------------------------------------------------------------------
Error : "Value cannot be null. (Parameter 'name')"
StackTrace : "   at System.Globalization.RegionInfo..ctor(String name)
   at CookieService.CookieSvc.GetUserCountry() in C:\TFS\4 identity-work bootstrap5 hebrew\busoftBase\CookieService\CookieSvc.cs:line 90"
InnerException : null
Source : "System.Private.CoreLib"
ActionId : "5eb3a7de-7216-4a3b-ba08-31df36a6f848"
ActionName : "CMS_CORE_NG.Areas.Admin.Controllers.AccountController.Login (CMS_CORE_NG)"
RequestId : "8000008e-0005-ff00-b63f-84710c7967bb"
RequestPath : "/Admin/Account/Login"
SpanId : "|4383733a-45e77c26e6fd7ea9."
TraceId : "4383733a-45e77c26e6fd7ea9"
ParentId : ""
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-4GF5G0T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19041.0"
Version : "3.1.11"
UserName : "ruti"
ProcessId : 10880
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 18:58:23 +03:00 | Level - Error |
----------------------------------------------------------------------
RequestId : "80000093-0004-fe00-b63f-84710c7967bb"
RequestPath : "/Admin"
SpanId : "|4383736a-45e77c26e6fd7ea9."
TraceId : "4383736a-45e77c26e6fd7ea9"
ParentId : ""
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-4GF5G0T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19041.0"
Version : "3.1.11"
UserName : "ruti"
ProcessId : 10880
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 18:58:26 +03:00 | Level - Error |
----------------------------------------------------------------------
Error : "Value cannot be null. (Parameter 'name')"
StackTrace : "   at System.Globalization.RegionInfo..ctor(String name)
   at CookieService.CookieSvc.GetUserCountry() in C:\TFS\4 identity-work bootstrap5 hebrew\busoftBase\CookieService\CookieSvc.cs:line 90"
InnerException : null
Source : "System.Private.CoreLib"
ActionId : "3683f267-a857-493f-ab11-1a085d2525a5"
ActionName : "CMS_CORE_NG.Areas.Admin.Controllers.AccountController.Login (CMS_CORE_NG)"
RequestId : "80000099-0004-fe00-b63f-84710c7967bb"
RequestPath : "/Admin/Account/Login"
SpanId : "|43837372-45e77c26e6fd7ea9."
TraceId : "43837372-45e77c26e6fd7ea9"
ParentId : ""
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-4GF5G0T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19041.0"
Version : "3.1.11"
UserName : "ruti"
ProcessId : 10880
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 20:06:06 +03:00 | Level - Error |
----------------------------------------------------------------------
RequestId : "80000032-0004-f800-b63f-84710c7967bb"
RequestPath : "/Admin/EmailSettings"
SpanId : "|4383749e-45e77c26e6fd7ea9."
TraceId : "4383749e-45e77c26e6fd7ea9"
ParentId : ""
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-4GF5G0T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19041.0"
Version : "3.1.11"
UserName : "ruti"
ProcessId : 10880
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 21/04/2021 20:06:23 +03:00 | Level - Error |
----------------------------------------------------------------------
Error : "Value cannot be null. (Parameter 'name')"
StackTrace : "   at System.Globalization.RegionInfo..ctor(String name)
   at CookieService.CookieSvc.GetUserCountry() in C:\TFS\4 identity-work bootstrap5 hebrew\busoftBase\CookieService\CookieSvc.cs:line 90"
InnerException : null
Source : "System.Private.CoreLib"
ActionId : "d1768776-032d-4c8a-b3c4-5b98f3d9402b"
ActionName : "CMS_CORE_NG.Areas.Admin.Controllers.AccountController.Login (CMS_CORE_NG)"
RequestId : "8000005c-0002-fe00-b63f-84710c7967bb"
RequestPath : "/Admin/Account/Login"
SpanId : "|438374ae-45e77c26e6fd7ea9."
TraceId : "438374ae-45e77c26e6fd7ea9"
ParentId : ""
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-4GF5G0T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19041.0"
Version : "3.1.11"
UserName : "ruti"
ProcessId : 10880
ProcessName : "iisexpress"
---------------------------------------------------------------------------
